internal/rbd/mirror.go

```diff
@@ -84,6 +88,35 @@ func (ri *rbdImage) promoteImage(force bool) error {
 	return nil
 }
 
+// forcePromoteImage promotes image to primary with force option with 1 minute
+// timeout. If there is no response within 1 minute,the rbd CLI process will be
+// killed and an error is returned.
+func (rv *rbdVolume) forcePromoteImage(cr *util.Credentials) error {
```

Comment:
@idryomov if force promote is rolling back a snapshot, then would it take time to respond? and if so, terminating it early would continue the operation or restart fresh?

Reply:
@ShyamsundarR this is client termination right, will it have any impact on the server-side? but good to get it confirmed :+1: